teach_base/lesson_01/02_triangle_square.rb

```diff
@@ -0,0 +1,17 @@
+=begin
+Площадь треугольника. Площадь треугольника можно вычилсить,
+ зная его основание (a) и высоту (h) по формуле: 1/2*a*h. 
+ Программа должна запрашивать основание и высоту треуголиника и возвращать его площадь.
+=end
+
+puts "Please input height of your triangle!"
+height = gets.chomp.to_f
+
+puts "Please input base of your triangle!"
+base = gets.chomp.to_f
+
+def tri_square (h, b)
+  0.5 *  h * b 
```

Comment:
Don't use extra spaces between math operators.
https://github.com/rubocop-hq/ruby-style-guide#spaces-and-operators